### PR TITLE
Purity-based QCD estimate for photon CR

### PR DESCRIPTION
**Pull request by AndreasAlbert**

### CODE REVIEW

**Comment by AndreasAlbert** [comment]
@alpakpinar a description of the inputs to this is an updated version of the same slides I sent you about the trigger

File: bucoffea/monojet/monojetProcessor.py [hunk 1]
```diff
@@ -527,6 +530,16 @@ def ezfill(name, **kwargs):
             ezfill('dphijm',             dphi=df["minDPhiJetMet"][mask],    weight=region_weights.weight()[mask] )
             ezfill('dphijr',             dphi=df["minDPhiJetRecoil"][mask],    weight=region_weights.weight()[mask] )
 
+            # Photon CR data-driven QCD estimate
+            if df['is_data'] and re.match("cr_g.*", region) and re.match("(SinglePhoton|EGamma).*", dataset):
+                w_imp = photon_impurity_weights(photons[leadphoton_index].pt, df["year"])
+                output['recoil'].fill(
+                                    dataset=data_driven_qcd_dataset(dataset),
+                                    region=region,
+                                    mjj=df["recoil_pt"][mask],
```
Comment (alpakpinar) [inline bucoffea/monojet/monojetProcessor.py]: The only point I'm confused is in this line actually, I'm not sure why you are filling an axis named "mjj" with "recoil_pt". Does this histogram have a mjj axis, or should it be something like 
'''
recoil=df["recoil_pt"][mask]
'''

**Review by alpakpinar — COMMENTED** [review]
Everything seems great, except I had one question on monojet processor, thanks!

File: bucoffea/monojet/monojetProcessor.py (hunk 1 above)
Comment (AndreasAlbert) [reply bucoffea/monojet/monojetProcessor.py]: Thanks for catching that. Indeed, it should have been `recoil=...`

**Pull request merged**